src/qililab/instruments/qblox/qblox_qcm.py

```diff
@@ -35,3 +36,11 @@ class QbloxQCMSettings(QbloxModule.QbloxModuleSettings):
         """Contains the settings of a specific pulsar."""
 
     settings: QbloxQCMSettings
+
+    def is_awg(self) -> bool:
+        """Returns True if instrument is an AWG."""
+        return True
+
+    def is_adc(self) -> bool:
+        """Returns True if instrument is an ADC."""
+        return False
```

Comment:
I don't understand why in some cases you have False here despite its description

Reply:
`is_awg` is True if the instrument contains an Arbitrary Waveform Generator, so in the case of QCM it is True.
`is_adc` is True if the instrument contains an Analog to Digital Converter, so in the case of QCM it is False.

The default of both in the parent class `Instrument` is False. So actually I can remove this overload.

Reply:
Ok now I understand, I misunderstood the functions and the documentation thinking it worked the other way around (you set the is_....() and then it updates your machine info), now it makes sense.